plugin.video.mediathek/mediathek/arte.py

```diff
@@ -195,15 +163,45 @@ def buildVideoEntry(self, jsonObject):
       subTitle = unicode(jsonObject["subtitle"]);
     else:
       subTitle = None;
-    detail = unicode(jsonObject["teaser"]);
-    picture = None;
-    for pictureItem in jsonObject["thumbnails"]:
-      if(picture is None or picture["width"]<pictureItem["width"]):
-        picture = pictureItem;
-    link=self.jsonLink%jsonObject["id"];
-    pubDate = time.strptime(jsonObject["scheduled_on"],"%Y-%m-%d");
-    duration = jsonObject["duration"]
-    self.gui.buildVideoLink(DisplayObject(title,subTitle,picture["url"],detail,link,"JsonLink", pubDate,duration),self,0);
+    if("teaser" in jsonObject):
+      detail = unicode(jsonObject["teaser"]);
+    else:
+      detail = "";
+
+    pictures = None;
+    pictureUrl = None;
+    if("thumbnails" in jsonObject):
```

Comment:
Just a comment here. You could use pictures = jsonObject.get("thumbnails") in place of the if statement. The second parameter of the .get() is the value to return if the index isn't found, which defaults to None.